tests/toolkit/matl_buy_policy_tests.cc

```diff
@@ -486,7 +486,7 @@ TEST_F(MatlBuyPolicyTests, NormalActiveDormant) {
 TEST_F(MatlBuyPolicyTests, MixedActiveDormant) {
   using cyclus::QueryResult;
   
-  boost::shared_ptr<NormalIntDist> a_dist = boost::shared_ptr<NormalIntDist>(new NormalIntDist(5, 1, 0, 1e299));
+  boost::shared_ptr<NormalIntDist> a_dist = boost::shared_ptr<NormalIntDist>(new NormalIntDist(5, 1, 0, std::numeric_limits<int>::max()));
```

Comment:
Was this necessary somehow, or is this just some cleanup?

Reply:
Trying to get cycamore working on macos and running into issues with the random number generator again.  We use 1e299 many places across the codebases.  Is it just intended to represent a really large number or is there another justification to this that I'm not aware of?  If the former is true should we make a separate issue/PR to follow best practices?

Reply:
Yes. I agree. We could also declare a `cyclus_large` somewhere and set it to the max int, and use that to be both descriptive and allow a single point of changing it